src/shivani/Shop.java

```diff
@@ -0,0 +1,72 @@
+package shivani;
+
+public class Shop {
+	int stockOfMaggie = 50;
+	int stockOfDosa = 43;
+	int stockOfPouches = 39;
+	int stockOfPanipuri = 43;
+	int stockOfMasala = 73;
+	int item1Q, item2Q, item3Q, item4Q, item5Q;
+	String customer;
+
+	void displayOOS() {
+		if (stockOfMaggie == 0 || stockOfMaggie < item1Q)
+
+			System.out.println(" maggi is Out of Stock");
+		else
+			System.out.println("Maggi is availabe in stock Item: " + stockOfMaggie);
+		if (stockOfDosa == 0 || stockOfDosa < item2Q)
+			System.out.println("Dosa is Out of Stock");
+		else
+			System.out.println("Dosa is available is stock Item:" + stockOfDosa);
+		if (stockOfPouches == 0 || stockOfPouches < item3Q)
+			System.out.println("Pouches is Out of Stock");
+		else
+			System.out.println("Pouches is available in stock, Item: " + stockOfPouches);
+		if (stockOfPanipuri == 0 || stockOfPanipuri < item4Q)
+			System.out.println("Panipuri is Out of Stock");
+		else
+			System.out.println("PaniPuri is avialable in stock,Item: " + stockOfPanipuri);
+		if (stockOfMasala == 0 || stockOfMasala < item5Q)
+			System.out.println("Masala is Out of Stock");
+		else
+			System.out.println("Masala is available in Stock , item :" + stockOfMasala);
+
+	}
+
+	public void shopping(int item1Q, int item2Q, int item3Q, int item4Q, int item5Q) {
+		if (stockOfMaggie == 0)
```

Comment:
condition should be if (stockOfMaggie > intem1Q) then you should permit to purchase items, else should show the available stocks